src/services/ShSimulatorService.tsx

```diff
@@ -56,6 +59,12 @@ export type JobResults = {
 	jobId: string;
 } & ResponseGetJobResults;
 
+export type SpecificEstimator = {
```

Comment:
what is the purpose of this type ? name suggest single object (single estimator), while the estimators seems like list

Reply:
to handle specific response of single Estimator

Reply:
then why do we need `estimators` (plural) inside ?